JitsiConference.js

```diff
@@ -843,13 +843,16 @@ function (jingleSession, jingleOffer, now) {
     }
     // add info whether call is cross-region
     var crossRegion = null;
-    if (window.jitsiRegionInfo)
+    if (window.jitsiRegionInfo) {
         crossRegion = window.jitsiRegionInfo["CrossRegion"];
-    Statistics.analytics.sendEvent("session.initiate",{
+    }
+    Statistics.analytics.sendEvent(
+        "session.initiate",
+        {
```

Comment:
Nope, we use different braces alignment in JS, the opening one should be on the previous line